spec/presenters/hyku/manifest_enabled_work_show_presenter_spec.rb

```diff
@@ -37,7 +37,18 @@
 
     it "returns a hash containing the rendering information" do
       work.rendering_ids = [work.file_sets.first.id]
-      expect(subject).to include(:sequence_rendering)
+      expect(subject).to be_an Array
+    end
+  end
+
+  describe "#manifest metadata" do
+    subject do
+      presenter.manifest_metadata
+    end
+
+    it "returns a hash containing the metadata" do
```

Comment:
There should probably be a test for the case of multi-valued fields.

Reply:
Update incoming ...